### ISSUE TITLE
Argument #1 ($user) must be of type Lunar\Hub\Models\Staff

### ISSUE THREAD

**Issue by raliqala** [lunarphp/lunar#1131]
- Lunar version: 0.3.1
- Laravel Version: 10.10
- PHP Version: 8.1
- Database 10.4.28-MariaDB:

### Expected Behaviour:  
I want to be able to login using users in the default users table as well.

### Actual Behaviour:
When I try to login on http://127.0.0.1:8000/login -> http://127.0.0.1:8000/dashboard after installing lunar, it takes me to dashboard as it should but now I get the following error  
```
Lunar\Hub\AdminHubServiceProvider::Lunar\Hub\{closure}(): Argument #1 ($user) must be of type Lunar\Hub\Models\Staff, App\Models\User given, called in C:\Users\john\Documents\ecom\example-ecom\vendor\laravel\framework\src\Illuminate\Auth\Access\Gate.php on line 575
```  
 when I login to /hub it works but on the other side it stopped working.

I'm using spatie/permissions, jetstram with livewire --teams.

### Steps To Reproduce:
I'm not sure about reproducing but you can:  

- Create a laravel project
- install jetstream livwire --teams
- Then install lunar following the docs

**Comment by glennjacobs** [comment]
That isn't supported currently. You must use the Staff model.

**Comment by raliqala** [comment]
ok thanks. is it possible to use luner with spatie/pamissions

**Comment by raliqala** [comment]
I created a fresh install and it works I can now login here http://127.0.0.1:8000/teams/1 and here http://127.0.0.1:8000/hub
![image](https://github.com/lunarphp/lunar/assets/30502407/dc86624e-f594-4e79-8632-6234e39b9a4f)

**Comment by raliqala** [comment]
I'm so happy  
I noticed there are no fields fo seo but I can just contribute them and metafields

**Comment by glennjacobs** [comment]
So it seems I misunderstood your bug report. It was a duplicate of #1129 which was resolved in 0.3.2.

Spatie Permissions is coming in the next release, see PR #1092 

And yes, you can add as many custom attributes as you like to Products (and other things). See https://docs.lunarphp.io/core/reference/attributes.html
